lib/OcrProcessors/OcrMyPdfBasedProcessor.php

```diff
@@ -139,6 +149,11 @@ private function getCommandlineArgs(WorkflowSettings $settings, GlobalSettings $
 			$args[] = '-j ' . $processorCount;
 		}
 
+		// Save recognized text in tempfile
+		if ($this->recognizedTextFile) {
```

Comment:
- [ ] A testcase in `PdfOcrProcessorTest.php` should be added, checking the sidecar-file behaviour. "The `--sidecar` parameter should be added if temporary file can be created"

Reply:
Hey @R0Wi ,
could you give me a hand with these test cases, I've tried quite some time now but don't seem to get them running right..

Reply:
Sure, thank's for your effort. Would it be an idea if i add you as contributor and we work together at your branch?